.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         python-version: 3.11
 
     - name: Install requirements
-      run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
+      run: python -m pip install -r requirements/base.txt -r requirements/dev.txt
```

Comment:
Per other comments, such implementation is unidiomatic. Use explicit `-c` and `-r` for the proper environment, without referencing other unrelated dependency groups.